addons/sale/models/sale_order_line.py

```diff
@@ -1157,10 +1157,15 @@ def _validate_analytic_distribution(self):
 
     #=== CORE METHODS OVERRIDES ===#
 
+    def _get_partner_display(self):
+        self.ensure_one()
+        if self.order_partner_id.parent_id:
```

Comment:
I think we should instead take the ancestor instead of just the parent because the parent could be the manager of the customer and not his company for instance. 🤔